docs/index.md

```diff
@@ -52,8 +52,8 @@ steampipe plugin install zoom
 
 | Item | Description |
 | - | - |
-| Credentials | [Create an SDK App](https://marketplace.zoom.us/docs/guides/build/sdk-app) to get the API key and secret. |
-| Permissions | SDK / JWT apps can access all Zoom APIs. |
+| Credentials | [Sign in](https://zoom.us/signin#/login) to Zoom to generate credentials. [Create a JWT App](https://marketplace.zoom.us/develop/create) to get the API key and secret. |
```

Comment:
@karanpopat As per https://developers.zoom.us/docs/platform/build/jwt-app/, JWT apps have been deprecated and will eventually not work with authorization. We can update this link here, but do you know if the new Server-to-Server OAuth app (or the OAuth app, but seems like Server-to-Server is more of what we'd use for the plugin) works with the current SDK we're using?

If not, we may need to add in code to get the OAuth creds ourselves. For instance, in https://devforum.zoom.us/t/jwt-app-vs-server-to-server-oauth-app/74499, there is some sample code for JS, but the same principles could be applied to this plugin (one notable point is there are no refresh tokens).